observation/keycloak/install.sh

```
@@ -0,0 +1,29 @@
+#!/bin/sh
+## Script to install Keycloak for Rancher
+## Usage: ./install.sh <iam_host_name> [kube_config_file]
+## iam_host_name: Example iam.mosip.net
+
+if [ $# -lt 1 ]; then
+  echo "Usage: ./install.sh <iam_host_name> [kube_config_file]"; exit 1
+fi
+if [ $# -ge 2 ]; then
+  export KUBECONFIG=$2
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)